### ISSUE TITLE
external display not working

### ISSUE THREAD

**Issue by KenWuqianghao** [tobhe/ubuntu-asahi#124]
external display thru type c and hdmi doesn't seem to be working, any pointers on that?

**Comment by tobhe** [comment]
Hey @KenWuqianghao, display over USB-C is not yet working, this is a known limitation. It is listed as "DP Alt Mode" here in the [Asahi feature support table](https://github.com/AsahiLinux/docs/wiki/M2-Series-Feature-Support).

**Comment by LazerKnown** [comment]
so, if i use hdmi on a usb-c dongle, will it work??? is there any way to yse an external display?

**Comment by tobhe** [comment]
No, usb-c to HDMI will not work. The only thing that works is the actual HDMI port on some pro laptops (this is not a Ubuntu limitation but true for all Asahi variants).

**Comment by LazerKnown** [comment]
im sad, im using this os on macbook air m2, IT DOES NOT HAVE ANY HDMI PORT

**Comment by leon-v** [comment]
Where can we support the development of getting external USB-C/TB displays working?

**Comment by tobhe** [comment]
The best place to contribute financially would be the [Asahi Linux open collective](https://opencollective.com/asahilinux). We as a downstream directly profit from all the work done in Asahi.

**Comment by bharambetejas** [comment]
I got the display working on Macbook air M2 using the fairydust kernel. Try using the scripts here: https://github.com/bharambetejas/asahi-fairydust-display

**Comment by mdajminpradhan** [comment]
> I got the display working on Macbook air M2 using the fairydust kernel. Try using the scripts here: https://github.com/bharambetejas/asahi-fairydust-display

man, thank you soooooo much for sharing

**Comment by vparmar-art** [comment]
Does the fairydust setup need a display port monitor? Can't this work on hdmi monitors?

**Comment by adho08** [comment]
Does the fairydust kernel have any limitations? If no, why does ubuntu-asahi not ship with this kernel by default?

**Comment by CrT0701** [comment]
Are there any updates regarding this topic?
